### DISCUSSION TITLE
Pomocy

### DISCUSSION THREAD

**Discussion by Roladka897** [cc-tweaked/CC-Tweaked#1674] [Q&A]
Jak drukować w drukarce? Nie mogę wytworzyć atramentu ani nie wiem jak to zrobić pomocyyy

**Comment by Wojbie** [comment]
Hey. Czy przypadkiem próbujesz wykorzystać [Torbiel z atramentem (Ink Sac)](https://minecraft.wiki/w/Ink_Sac) zamiast [Czarny barwnik (Black Dye)](https://minecraft.wiki/w/Black_Dye) ? Barwniki zostały rozdzielone w oddzielny item kilka wersji minecrafta temu. Barwniki powinny bez problemu wejść w slot w drukarce na atrament.

Hello. Any chance you are trying to use Ink Sac instead of Black Dye? Dye items got split into separate items few minecraft versions ago. Dyes should fit the ink slot of printer without any issues.
